Cargo.toml

```diff
@@ -39,13 +39,16 @@ unstable = ["std", "broadcaster", "futures-timer"]
 attributes = ["async-attributes"]
 std = [
   "crossbeam-utils",
-  "futures-core",
   "futures-io",
   "memchr",
   "once_cell",
-  "pin-project-lite",
   "pin-utils",
   "slab",
+  "no-std",
+]
+no-std = [
```

Comment:
I believe `alloc` is a better feature name (futures, crossbeam, serde, etc. use this name.).

Reply:
That's right. I changed to `alloc`